app/controllers/contribs_controller.rb

```diff
@@ -3,6 +3,8 @@ class ContribsController < PartipsController
   def new
     super
 
+    # Interest.exists?(params[:in_reply_to])
+    # aber: per bspw. find_by oder find_by_id könnte man sich auch den zusätzlichen DB-Call sparen
     if Interest.exists?(:id => params[:in_reply_to])
```

Comment:
Actually we can get rid of the entire if check, because returning `nil` is just fine.